matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -961,42 +965,61 @@ var PhenoTips = (function (PhenoTips) {
         return td;
     },
 
-    _getNotified : function(record)
+    _getContact : function(record)
     {
-        var td = '<td style="text-align: center" name="contacted">';
+        var td = '<td style="text-align: center" name="contact">';
+        if (record.notified) {
+            td += this._CONTACTED_LABEL;
+        } else {
+            td += this._getContactButton(record);
+        }
+        td += '</td>';
+        return td;
+    },
+
+    _getContactButton : function(record)
+    {
+        var button = '';
         var accessAboveEdit = record.matched.access == "edit"   || record.reference.access == "edit"
                            || record.matched.access == "owner"  || record.reference.access == "owner"
                            || record.matched.access == "manage" || record.reference.access == "manage";
-        if (!this._isAdmin && accessAboveEdit && (!record.matched.pubmedIDs || record.matched.pubmedIDs.size() == 0) || (!record.reference.pubmedIDs || record.reference.pubmedIDs.size() == 0)) {
+        var oneCaseOrMoreUnsolved = (!record.matched.pubmedIDs || record.matched.pubmedIDs.size() == 0) || (!record.reference.pubmedIDs || record.reference.pubmedIDs.size() == 0);
```

Comment:
Not directly related to this issue, but seems to be a regression where we allow contacting patient that are solved